src/scripts/clipperUI/components/pdfClipOptions.tsx

```diff
@@ -191,8 +198,14 @@ class PdfClipOptionsClass extends ComponentBase<PdfClipOptionsState, ClipperStat
 		);
 	}
 
-	getEnabledAttachmentCheckbox(): any {
+	getAttachPdfCheckbox(clickable: boolean): any {
+		return clickable ? this.getEnabledAttachmentCheckbox() : this.getDisabledAttachmentCheckbox();
+	}
+
+	// TODO: Programmaticaly remove the enableInvoke so that we can collapse the attachmentCheckbox functions into one
```

Comment:
Good ideas! I will implement them and then verify that they work. I am a bit scared that virtualDOM won't pick up attaching/detaching of handlers if I do this, but this will be a good chance to find out.